MonoGame.Framework/Audio/SoundEffectInstance.XAudio.cs

```diff
@@ -39,6 +39,12 @@ private void PlatformApply3D(AudioListener listener, AudioEmitter emitter)
             e.CurveDistanceScaler = SoundEffect.DistanceScale;
             e.DopplerScaler = SoundEffect.DopplerScale;
             e.ChannelCount = _effect._format.Channels;
+            //stereo channel
+            if(e.ChannelCount > 1 )
+            {
+                e.ChannelRadius = 0;
+                e.ChannelAzimuths = new float[] { 0f, 0f };
+            }
```

Comment:
This looks like it could be the right path here, but we need some fixes.

First this PR should be rebuilt against a clean branch from `develop`.  There are two other non-related changes in this PR that need to be removed.

Second we should not be allocating a `float[]` over and over here.  We should allocate one static array and just assigning it here.

Third I am unsure if 0 is the correct values for the radius or azimuths.  It could be correct, it could be wrong.  We should do a little testing against XNA and make sure 0 here sounds correct to regular XNA behavior.

Other than that it seems like you're on the right track.